website_docs/getting-started.md

```diff
@@ -3,552 +3,940 @@ title: "Getting Started"
+                }
+        ],
+        "InstrumentationLibrary": {
+                "Name": "diceroller-service",
+                "Version": "",
+                "SchemaURL": ""
+        }
+}
+```
 
-# Putting It All Together
+</details>
+
+You'll find that the `dice-roller` span has information about a parent span, and
+the ID of that parent span matches the span created by automatic
+instrumentation.
+
+## Send traces to an OpenTelemetry Collector
+
+The [OpenTelemetry Collector](/docs/collector/getting-started/) is a critical
+component of most production deployments. Some examples of when it's beneficial
+to use a collector:
+
+* A single telemetry sink shared by multiple services, to reduce overhead of
+  switching exporters
+* Aggregating traces across multiple services, running on multiple hosts
+* A central place to process traces prior to exporting them to a backend
+
+Unless you have just a single service or are experimenting, you'll want to use a
+collector in production deployments.
+
+### Configure and run a local collector
+
+First, write the following collector configuration code into the `/tmp/`
+directory on your local machine:
+
+```yaml
+# /tmp/otel-collector-config.yaml
+receivers:
+    otlp:
+        protocols:
+            grpc:
+            http:
+exporters:
+    logging:
+        loglevel: debug
+processors:
+    batch:
+service:
+    pipelines:
+        traces:
+            receivers: [otlp]
+            exporters: [logging]
+            processors: [batch]
+```
 
-You should now have a working application that produces trace telemetry data! Give it a try.
+Then in a separate terminal window, run the docker command to acquire and run
+the collector based on this configuration:
 
-```sh
-$ go run .
-What Fibonacci number would you like to know:
-42
-Fibonacci(42) = 267914296
-What Fibonacci number would you like to know:
-^C
-goodbye
+```
+docker run -p 4318:4318 \
+    -v /tmp/otel-collector-config.yaml:/etc/otel-collector-config.yaml \
+    otel/opentelemetry-collector:latest \
+    --config=/etc/otel-collector-config.yaml
 ```
 
-A new file named `traces.txt` should be created in your working directory. All the traces created from running your application should be in there!
+You will now have an OpenTelemetry Collector instance running locally.
 
-# (Bonus) Errors
+### Modify the code to export spans via OTLP
 
-At this point you have a working application and it is producing tracing telemetry data. Unfortunately, it was discovered that there is an error in the core functionality of the `fib` module.
+The next step is to modify the code to send spans to the Collector via OTLP
+instead of the console. Change the code to create an OTLP exporter, replacing
+the console exporter from before with an OTLP HTTP exporter that talks to the
+local endpoint the collector is listening on.
 
-```sh
-$ go run .
-What Fibonacci number would you like to know:
-100
-Fibonacci(100) = 3736710778780434371
-# …
+```go
+exp, err :=
```

Comment:
this statement should match with the full sample code (use `exporter, err` rather than `exp, err`)

Reply:
further nit; I'd probably have the context import in `newTraceProvider` from the jump? also call out in the text that you'll need to add the appropriate imports here as well.